open_city_profile/static/open-city-profile/gdpr-api-openapi.yaml

```diff
@@ -0,0 +1,293 @@
+openapi: 3.0.1
+info:
```

Comment:
I check which versions are possible to set here and found this link https://en.wikipedia.org/wiki/OpenAPI_Specification
3.0.1 version is 5 years old.
I tried to set version to 3.1.0 and run it locally but I was shown Error that version must be 3.0.X
So, I think OpenAPI version has to be set to 3.0.3 for now.

Reply:
Age of a specification doesn't really matter. It can be decades old, if it's still valid and usable.

https://github.com/OAI/OpenAPI-Specification/releases

OpenAPI version 3.1 has some breaking changes compared to 3.0 so this document might need some changes in order to be compliant.

All the 3.0.x versions are identical when it comes to what the specification specifies, the differences are in wording etc. of the specification. In that sense it doesn't make a difference what patch version of 3.0.x series we have here.

I can still change the OpenAPI version to 3.0.3 though, because why not :smiley:

Reply:
Changed.